src/components/Tag/Tag.module.css

```diff
@@ -48,7 +48,6 @@
  */
 .tag--neutral {
   --tag-primary-color: var(--eds-theme-color-text-neutral-default);
-  /* TODO(brand-tokens): use background-neutral-medium when new colors land */
```

Comment:
verified in https://www.figma.com/file/Ff7hJoSvXX9hkAmIklnsT2/EDS---UI-Components-(Main)?node-id=1195%3A8117 that we do want to use `background-neutral-subtle` after all